docs/lib/datastore/getting-started.md

```diff
@@ -9,44 +9,161 @@ Amplify DataStore provides a programming model for leveraging shared and distrib
 
 <amplify-callout>
 
-DataStore can be used as a local-only data persistence mechanism and an AWS account is not required for that. However, if you wish to sync with the cloud it is recommended you [Install and configure the Amplify CLI](~/cli/start/install.md).
+**Note:** this allows you to start persisting data locally to your device with DataStore, even without an AWS account.
 
 </amplify-callout>
 
-## Generate models
+## Goal
+To setup and configure your application with Amplify DataStore and use it to persist data locally on a device.
 
-Modeling your data and *generating models* which are used by DataStore is the first step to get started. GraphQL is used as a common language across all supported platforms for this process, and is also used as the network protocol when syncing with the cloud. GraphQL is also what powers some of the features such as [Automerge in AppSync](https://docs.aws.amazon.com/appsync/latest/devguide/conflict-detection-and-sync.html#automerge).
+## Prerequisites
+
+<inline-fragment platform="js" src="~/lib/datastore/fragments/js/getting-started/10_preReq.md"></inline-fragment>
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/10_preReq.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/10_preReq.md"></inline-fragment>
+
```

Comment:
Could be that React Native case is not be completely clear here?

Reply:
Is there a special setup process for RN?